scripts/interactions.js

```diff
@@ -101,11 +112,27 @@ function keypress(e){
 	if(unicode === 108){ // l
 		if(graph.selection[0] && graph.selection[1])
 		{
+			var selection_src_id, selection_tgt_id;
+
+			if("data" in graph.selection[0] && "data" in graph.selection[1])
+			{
+				selection_src_id = graph.selection[0].data._id;
+				selection_tgt_id = graph.selection[1].data._id;
+			}
+			else
+			{
+				selection_src_id = graph.selection[0]._id;
+				selection_tgt_id = graph.selection[1]._id;
+			}
 			damas.create_rest({
-				src_id: graph.selection[0].data._id,
-				tgt_id: graph.selection[1].data._id }, function(node){
+				src_id: selection_src_id,
+				tgt_id: selection_tgt_id }, 
+				function(node){
 					graph.newEdge(node);
-			});
+				}
+			);
+			alert('LINK CREATED! src_id: '+ selection_src_id+', tgt_id: '+selection_tgt_id );
+			graph.unselectAll();
```

Comment:
see the comment above, the selection array should contain abstract nodes so the id of a node is available in node._id directly - else we will test data._id existence forever